builtin/game/anticheat.lua

```diff
@@ -0,0 +1,288 @@
+core.registered_anticheat_objects = {}
+
+local current_anticheat_objects = {}
```

Comment:
missing a comment explaining what these are for
they seem to store state(?) for each player